Makefile

```diff
@@ -47,6 +47,7 @@ postgres:
 
 
 lint:
+	(cd ./internal/tools/querylinter/cmd; go build -o ./querylinter.so -buildmode=plugin .)
```

Comment:
Any way we can get around building this every time we run linting, or is it pretty lightweight?